### ISSUE TITLE
FR: clone

### ISSUE THREAD

**Issue by hemanth** [tj/git-extras#434]
Something on the lines of [lazyclone](http://stackoverflow.com/questions/12058705/a-better-way-to-do-git-clone?answertab=votes#tab-top) will be useful?

//cc @spacewander

**Comment by spacewander** [comment]
You mean we can write a command called `git lazyclone` with this skill?

**Comment by hemanth** [comment]
No, another command in `git-extars` like `git-clone`

**Comment by spacewander** [comment]
@hemanth 
What should we name it? 
Is `git-get` short enough for lazy guy?

**Comment by hemanth** [comment]
heh heh nice, yeah :+1:

**Comment by spacewander** [comment]
Terribly, I just figured out we can't change user's pwd in a shell script, unless we put the function in  `.bashrc`

**Comment by hemanth** [comment]
well, we are just cloning the repo here ain't we?

**Comment by spacewander** [comment]
Yes, cloning the repo may be the only thing we can do... Since change the pwd of parent shell(used in a tty) is impossible in a subshell(running our script), they are difference processes.
